entries/u/ukraine.com.ua.json

```
@@ -1,15 +1,13 @@
 {
-  "Ukraine": {
+  "Hosting Ukraine": {
     "domain": "ukraine.com.ua",
-    "url": "https://www.ukraine.com.ua/",
-    "img": "ukraine.com.ua.png",
     "tfa": [
       "sms",
       "call",
       "totp",
       "custom-hardware"
```

```suggestion
      "custom-software"
```
I can't find any reference to hardware-based 2FA on the page, but they do seem to mention 2FA via a Telegram bot? Maybe this should be updated to include that, along with a `custom-software` property:
```JSON
"custom-software": [
  "Telegram bot"
]
```